extras/dgoss/dgoss

```diff
@@ -33,7 +34,7 @@ run(){
 get_docker_file() {
     if docker exec "$id" sh -c "test -e $1" > /dev/null;then
         info "Copied '$1' from container to current directory"
```

Comment:
Got a chance to test it today, the error case is a bit unclear, but the fix is easy.

I think if this info line is changed to the GOSS_FILES_PATH param instead of "current directory" the user would have all the info they need.